setup.py

```diff
@@ -67,7 +67,7 @@ def run_tests(self):
     include_package_data=True,
     license = 'MIT',
     install_requires = [
-        'alembic',
+        'alembic<=1.6.5',
```

Comment:
Since 1.7 is the breaking version, why not use that as the limit? That way newer patch release to 1.6 would still be allowed.
```suggestion
        'alembic<1.7',
```